index.html

```diff
@@ -1037,9 +1037,12 @@ <h3>Identifiers</h3>
         <p>
 When expressing statements about a specific thing, such as a person, product,
 or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. Identifiers that others
```

Comment:
I think a simpler change that reflects the thinking of the group would be, "If present, identifiers that others ..."